spec/controllers/hyrax/collections_controller_spec.rb

```diff
@@ -1,130 +1,162 @@
 # frozen_string_literal: true
 RSpec.describe Hyrax::CollectionsController, clean_repo: true do
-  routes { Hyrax::Engine.routes }
-  let(:user)  { create(:user) }
-  let(:other) { build(:user) }
-
-  let(:collection) do
-    create(:public_collection_lw, title: ["My collection"],
-                                  description: ["My incredibly detailed description of the collection"],
-                                  user: user)
-  end
+  [false, true].each do |use_valkyrie|
```

Comment:
i'm a little uncertain about the need for this. much of the behavior in this test setup is already using valkyrie directly, and the internal behavior of the mixin should be equivalent however the test is setup, right?

what's the motivation for the duplicate test runs?

Reply:
The error with the missing attribute `member_of_collection_ids` on `Hyrax::PcdmCollection` was not uncovered by these tests.  There wasn't a failing test until `use_valkyrie=true`.

This is part of what prompted me to ask if we want to start migrating tests to only run with `Valkyrie` objects and `use_valkyrie=true` when possible.  In that case, the change here would be to only run the `use_valkryie=true` set of tests.

Reply:
i think that's right. this controller should function the same way however the objects were created. migrating the tests to setup with valkyrie seems like something we can just do, without any branching or iteration.

Reply:
Ok, I will drop the AF setup and only test with Valkyrie resources.

Reply:
@no-reply This change has been made.  The tests only use Valkyrie resources.